### PR TITLE
Update 2019-05-13-prestashop-1-7-6-0-beta-release.md

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

**Comment by ttoine** [comment]
needs review and pictures

File: news/_posts/releases/2019-05-13-prestashop-1-7-6-0-beta-release.md [hunk 1]
```diff
@@ -42,50 +42,54 @@ If you find regressions compared to previous versions, please create an issue on
...
-### New features & improvements
+### SEO Optimization
 
--   SEO optimization for combination products: all combinations now have the parent product’s URL as their canonical, accessing it no longer issues a redirection, displays the default combination [\#13009](https://github.com/PrestaShop/PrestaShop/pull/13009)
--   A “Price tax included” column has been added to the products list [\#11484](https://github.com/PrestaShop/PrestaShop/pull/11484)
--   A new “modern” template design & wording for transactional emails and a
-    new email template generation system[\#13004](https://github.com/PrestaShop/PrestaShop/pull/13004)
+SEO optimization for combination products:
+- All combinations now have the parent product’s URL as canonical,
+- Going the the parent product’s URL displays the default combination, there no redirection anymore
```
Comment (colinegin) [inline news/_posts/releases/2019-05-13-prestashop-1-7-6-0-beta-release.md]: Going to the parent product’s URL displays the default combination, there is no redirection anymore

File: news/_posts/releases/2019-05-13-prestashop-1-7-6-0-beta-release.md [hunk 2]
```diff
@@ -42,50 +42,54 @@ If you find regressions compared to previous versions, please create an issue on
...
-    -   An avatar image is now displayed
-- More Helpers cards and Empty states added to the BO
+-The **Design section of your back office** has been redesigned. Now you are able to have an overview of all design features (RTL, logos, favicon and theme) and manage them the way you want.
 
 ![More Helpers Cards in the BO](/assets/images/2019/05/1.7.6_beta_empty_states.jpg)
 
+- Improved employee profile menu in the back office
+  - Five new tabs added: Resources, Training, Find an expert, PrestaShop Marketplace and Help center
+  - Your avatar image is now displayed
+- Several back office design bugs have been fixed and made some integration improvements:
```
Comment (colinegin) [inline news/_posts/releases/2019-05-13-prestashop-1-7-6-0-beta-release.md:81]: Several back office design issues have been fixed or improved :

File: news/_posts/releases/2019-05-13-prestashop-1-7-6-0-beta-release.md [hunk 3]
```diff
@@ -42,50 +42,54 @@ If you find regressions compared to previous versions, please create an issue on
...
-    -   Fix label "Web only" [\#12872](https://github.com/PrestaShop/PrestaShop/pull/12872)
-    -   Fix quantity placeholder issue in the Stock management page [\#10199](https://github.com/PrestaShop/PrestaShop/pull/10199)
-    -   Fix image dimensions in Product Card view [\#10297](https://github.com/PrestaShop/PrestaShop/pull/10297)
-    -   Limit carrier image size on checkout page [\#12892](https://github.com/PrestaShop/PrestaShop/pull/12892)
-    -   Add modal-dialog-centered class to classic theme [\#10253](https://github.com/PrestaShop/PrestaShop/pull/10253)
 
+### Other improvements 
 
-### New & improved core modules 
+- New “modern” template design and  wording  for all transactional emails along with a  new email template generation system and a new back office page dedicated to email management (in Design > Email theme)
```
Comment (colinegin) [inline news/_posts/releases/2019-05-13-prestashop-1-7-6-0-beta-release.md]: seems like there is a double space between "with a" and "new"

**Comment by ttoine** [comment]
thanks @colinegin

**Pull request merged**